packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -766,6 +775,15 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   private _resetProperties() {
+    let colormap;
+    if (this.useCPURendering) {
+      colormap = 'Gray';
+    } else {
+      colormap = 'Grayscale';
+    }
```

Comment:
This is not correct. if you have RGB images, and you render in stack viewport, you don't want the reset to make them grayscale